### PR TITLE
[PREP-295] Add google scholar and more meta tags

### PR DESCRIPTION
**Pull request by hmoco**
## Purpose

 Add google scholar and more meta tags

## Ticket

https://openscience.atlassian.net/browse/PREP-295

### CODE REVIEW

File: app/routes/content.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
             const {origin} = window.location;
             const image = this.get('theme.logoSharing');
             const imageUrl = `${origin.replace(/^https/, 'http')}${image.path}`;
-
+            const d = new Date(preprint.get('dateCreated'));
```
Comment (brianjgeiger) [inline app/routes/content.js]: How about full word variables?

File: app/routes/content.js [hunk 2]
```diff
@@ -104,8 +104,17 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated')).toISOString()],
-                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()]
+                ['article:published_time', d.toISOString()],
+                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()],
+                ['citation_title', node.get('title')],
+                ['citation_description', node.get('description')],
+                ['citation_public_url', window.location.href],
```
Comment (brianjgeiger) [inline app/routes/content.js]: The API response should have a web url for this preprint, right? That seems a better choice than the location.

File: app/routes/content.js [hunk 3]
```diff
@@ -104,8 +104,17 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
...
+                ['citation_publication_date', `${d.getFullYear()}/${d.getMonth() + 1}/${d.getDate()}`],
+                ['citation_doi', preprint.get('doi')],
+                ['dc.title', node.get('title')],
+                ['dc.abstract', node.get('description')],
+                ['dc.identifier', window.location.href],
```
Comment (brianjgeiger) [inline app/routes/content.js]: Same here on getting this from the API if it's available. And should there be two `dc.identifier` fields?

Comment (hmoco) [inline app/routes/content.js]: Yes! A document can have multiple identifiers.

File: app/routes/content.js [hunk 4]
```diff
@@ -138,6 +153,35 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 )));
 
                 this.get('headTagsService').collectHeadTags();
+
+                preprint.get('provider').then(provider => {
```
Comment (binoculars) [inline app/routes/content.js]: @hmoco http://blog.ministryofprogramming.com/prevent-pyramids-with-flat-promise-chains/
tl;dr: return a promise from inside a then

File: app/routes/content.js [hunk 5]
```diff
@@ -100,12 +100,21 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
...
-                ['og:url', window.location.href],
+                ['og:url', preprint.get('links.html')],
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated')).toISOString()],
-                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()]
+                ['article:published_time', dateCreated.toISOString()],
+                ['article:modified_time', new Date(preprint.get('dateModified')).toISOString()],
+                ['citation_title', node.get('title')],
```
Comment (binoculars) [inline app/routes/content.js]: Can't mix order of tags. The order matters for the scrapers. The OGP tags must be sequential. Updating this in another PR with the redirect fix.
